tls-tester/src/com/axonivy/test/tls/TLSTest.java

```diff
@@ -155,33 +154,29 @@ private void loadClientTruststore()
 
   private void loadCustomClientKeystore()
   {
+    var ks = sslClientSettings.getKeyStore();
     TLSTestData data = new TLSTestData(TLSTestGroup.CLIENT_CUSTOM_KEYSTORE);
-    if (!sslClientSettings.useCustomKeyStore())
+    if (!ks.useCustomKeyStore())
     {
       data.addEntry("Custom client KeyStore set to NOT be used - skipping");
       data.setResult(2);
     } else
     {
-      customKeyStore = loadKeyStore(sslClientSettings.getKeyStoreFile(), sslClientSettings.getKeyPassword()
-          .toCharArray(), sslClientSettings.getKeyStoreType(), sslClientSettings.getKeyStoreProvider(),
+      customKeyStore = loadKeyStore(ks.getFile(), ks.getKeyPassword()
+          , ks.getType(), ks.getProvider(),
           "CustomKeyStore", data);
     }
     logs.add(data);
   }
 
   private void loadCustomClientTruststore()
   {
+    var ts = sslClientSettings.getTrustStore();
     TLSTestData data = new TLSTestData(TLSTestGroup.CLIENT_CUSTOM_TRUSTSTORE);
-    if (!sslClientSettings.useCustomTrustStore())
-    {
-      data.addEntry("Custom client TrustStore set to NOT be used - skipping");
-      data.setResult(2);
-    } else
-    {
-      customTrustStore = loadKeyStore(sslClientSettings.getTrustStoreFile(), sslClientSettings.getTrustStorePassword()
-          .toCharArray(), sslClientSettings.getTrustStoreType(), sslClientSettings.getTrustStoreProvider(),
-          "CustomTrustStore", data);
-    }
```

Comment:
the custom trust store seems to a be valid feature in the "tls-tester" environment. 
it's here to deal with a manually uploaded store: see the readme.md

Reply:
so I'd not remove this features right now ... but only when we integrate it into the core.

Reply:
as discussed in other PR; this is fine